modules/setting-up-topology-manager.adoc

```diff
@@ -14,6 +14,41 @@
 +
 ----
 # oc edit featuregate/cluster
+
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  annotations:
+    release.openshift.io/create-only: "true"
+  creationTimestamp: 2020-06-04T17:55:29Z
+  generation: 2
+  managedFields:
+  - apiVersion: config.openshift.io/v1
+    fieldsType: FieldsV1
+    fieldsV1:
+      f:metadata:
+        f:annotations:
+          .: {}
+          f:release.openshift.io/create-only: {}
+      f:spec: {}
+    manager: cluster-version-operator
+    operation: Update
+    time: 2020-06-04T17:55:29Z
+  - apiVersion: config.openshift.io/v1
+    fieldsType: FieldsV1
+    fieldsV1:
+      f:spec:
+        f:featureSet: {}
+    manager: oc
+    operation: Update
+    time: 2020-06-04T18:40:06Z
+  name: cluster
+  resourceVersion: "31004"
+  selfLink: /apis/config.openshift.io/v1/featuregates/cluster
+  uid: c63e9fd5-265f-400b-89ea-46cc9c975488
+spec:
+  featureSet: LatencySensitive
+
```

Comment:
I don't know what's the standard, but we have the option to use a simpler yaml with `oc apply -f`, which will just apply the delta. It could be simple as: (caution *UNTESTED*, for illustration purposes only)
```bash
$ cat enable-tm.yaml
apiVersion: config.openshift.io/v1
kind: FeatureGate
metadata:
  name: cluster
spec:
  featureSet: LatencySensitive
$ oc apply -f enable-tm.yaml
```
also: do we need to specify with more detail what happens if for whatever reason another feature gate was already enabled? (I think no, but worth doublechecking)

Reply:
This was the output from a deploy of openshift on the latest 4.5 nightly build. I just showed the contents to give an example of enabling the LatencySensitive feature, as there was some slight confusion about the case/name between `FeatureGate` and `featureSet`.

I would expect the openshift [featureGate](https://docs.openshift.com/container-platform/4.4/nodes/clusters/nodes-cluster-enabling-features.html) docs to have more details about specifying multiple featureGates. Took a look, but it is not there :-(

Fortunately, there is some additional [featureGate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) docs in kubernetes that specifies there can be multiple featureGates enabled, comma separated list.

Reply:
The confusion in the 4.4 docs was mainly from `oc describe featuregate/cluster` output in the example, which is `Feature Set:  LatencySensitive`.  It might be better to show `oc get featuregate/cluster -o yaml` output in the example so the user can see that they need to add under spec: `featureSet: LatencySensitive`.

Reply:
OK, let me take a look and see how that looks. Waiting for the cluster to spin up

Reply:
Agree, changed the example to show `oc get featuregate/cluster -o yaml. `

Reply:
And one last minor change I just pushed, change Step 2 to:

 `Add featureSet: LatencySensitive to the spec:`

Reply:
@vpickard Is this ready to merge? For 4.5 only?
I see a couple of editorial tweaks that I can clean up after.

Reply:
@mburke5678 Yes, should be good to go. I would think, merge to master, then cherry-pick to 4.5?